backend/associations/serializers/media.py

```diff
@@ -37,5 +38,5 @@ def urlize(request, url):
     if not url or url.startswith("http"):
         return url
 
-    host = request.get_host()
+    host = env.str("BACKEND_HOST")
```

Comment:
Why doesn't `request.get_host()` work?

Reply:
I think it is because I need host to be `localhost:port` but request.get_host() will only return `localhost`

Reply:
I think it does return  `localhost:port`

Reply:
Indeed ! In fact the issue was that the backend is running on port 8080 in production (for instance) but to reach it you have to reach nginx proxy which is port 1337 for instance.